response_operations_ui/controllers/iac_controller.py

```diff
@@ -0,0 +1,49 @@
+import logging
+
+import requests
+from structlog import wrap_logger
+
+from response_operations_ui import app
+from response_operations_ui.exceptions.exceptions import ApiError
+
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+def get_iac(iac):
```

Comment:
🎨 A little confusing naming here: `get_iac` has `iac` as a param and returns a JSON response.

Reply:
:+1: Will replace this name with `get_iac_details`